src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -51,6 +54,7 @@ const RenderListFeed = (props) => {
     navigateToLinkContextPage,
     getHeightFooter,
     getHeightHeader,
+    getTotalReaction,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getTotalReaction` has been added but it's not clear where this function is defined. Please ensure that it is imported or defined in the same file.
```diff
+    getTotalReaction,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->